app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -66,4 +74,34 @@ public static void setAdapter(RecyclerView recyclerView, RecyclerView.Adapter ad
         recyclerView.setAdapter(adapter);
     }
 
+    @BindingAdapter("playlist_adapter")
+    public static void setPlaylistAdapter(RecyclerView view, RecyclerView.Adapter adapter) {
+        view.setAdapter(adapter);
+        view.setItemAnimator(new DefaultItemAnimator());
+        view.addItemDecoration(
+                new DividerItemDecoration(view.getContext(), LinearLayoutManager.VERTICAL));
+    }
+
+    @BindingAdapter("settext")
+    public static void setNumberSong(TextView textView, int numberSong) {
+        textView.setText(numberSong + " "
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.